frontend/src/styles.css

```diff
@@ -1,10 +1,25 @@
 html,
 body,
 #app-root {
-  font-family: 'Source Sans 3', sans-serif;
+  font-family: 'Roboto', sans-serif;
```

Comment:
@dianaPrahoveanu-SL just fyi Mike's adding some of the theme logic here for his ticket 😄